src/scripts.js

```diff
@@ -344,9 +348,38 @@ function populatePantry() {
       <tr>
         <td>${ingredientData.name}</td>
         <td>${item.amount}</td>
+        <td class="button-column">
+          <button class="round-buttons">+</button>
+          <button class="round-buttons">-</button>
+        </td>
       </tr>`
   })
-  console.log(currentUser.name)
+}
+
+function populateDropdown() {
+  const ingData = ingredientsData.filter(ingredient => {
+    return typeof ingredient.name === 'string'
+  })
+  const sortedIngredients = ingData.sort((a, b) => {
+    if(((typeof a.name) === 'string') && ((typeof b.name) === 'string')) {
```

Comment:
I feel like this is pretty nifty. Perhaps a bit ugly, but it gets us where we needed since we're dealing with 'undefined' in the api.